src/selectors/entities/preferences.ts

```diff
@@ -89,14 +89,14 @@ export const getVisibleGroupIds = reselect.createSelector(
 export const getTeammateNameDisplaySetting = reselect.createSelector(
     getConfig,
     getMyPreferences,
-    (config, preferences) => {
+    getLicense,
+    (config, preferences, license) => {
+        const useAdminTemmateNameDisplaySetting = (license && license.LockTeammateNameDisplay === 'true') && config.LockTeammateNameDisplay === 'true';
```

Comment:
Should this be `useAdminTeammateNameDisplaySetting`? (typo)

Reply:
Done! Thanks